assets/overlays/aws-efs/generated/standalone/controller.yaml

```diff
@@ -200,6 +190,24 @@ spec:
         - mountPath: /etc/tls/private
           name: metrics-serving-cert
       hostNetwork: true
+      initContainers:
+      - command:
+        - sh
+        - -c
+        - |
+          cat <<EOF > /var/run/aws/auth/credentials
+          [default]
+          aws_access_key_id=$(cat /var/run/aws/keys/aws_access_key_id)
```

Comment:
Those files won't exist on STS cluster so it will create a file like:

```
[default]
aws_access_key_id=
aws_secret_access_key=
```

But as long as it does not overwrite STS config it should be fine I guess. Currently the config in STS is at `/var/run/secrets/aws/credentials`, with this patch it would be `/var/run/aws/keys/credentials`. Maybe the script could check the files first?

Reply:
Good point, thanks. I fixed this in my other PR for EBS, once that one merges I'll do the same here.